lib/Runtime/Library/JavascriptPromise.cpp

```diff
@@ -559,6 +559,159 @@ namespace Js
         return CreateThenPromise(promise, fulfillmentHandler, rejectionHandler, scriptContext);
     }
 
+    // Promise.prototype.finally as described in the draft of ES 2018 Section 25.4.5.3
```

Comment:
nit:  ECMA section numbers aren't immutable, but the anchors are considered to be.  Use `#sec-promise.prototype.finally` instead of a section number.

Reply:
I was copying what was already in the file on this one - most of the Promise methods have comments referring to section numbers per ES2015 (which are also mostly out of date now) if I'm going to fix this point assume I should fix all of them at the same time?

Reply:
Might as well fix this one now and follow up with the others 

---
In reply to: [167731238](https://github.com/Microsoft/ChakraCore/pull/4668#discussion_r167731238) [](ancestors = 167731238)

Reply:
If we're going to update the other section numbers, I'd like to make sure we also do an audit to make sure no behavior in the functions has changed between spec versions. Then we can be sure the implementation holds to the new spec version and update the spec reference at the same time. So, probably best to follow-up on the other methods.